js/options/options.js

```diff
@@ -387,6 +387,7 @@ let Options = (function(){
             return `<div class="country_parent">
             <span class='es_flag ${countryClass}'></span>
             <select class='regional_country'>${options}</select>
+            <button type="button" name="profile_custom_remove" class="custom-link__close"></button>
```

Comment:
Why did you give it that `name`?

Reply:
I think I copied it. I'll check and see if i can remove/change it.

Reply:
Removed. That shouldn't affect anything.